src/register/constants/pbaNumber.ts

```diff
@@ -19,43 +19,53 @@ export const pbaNumber = {
     ],
     groups: [
       {
-        input: {
-          label: {
-            text: 'PBA number 1 (optional)',
-            classes: 'govuk-label--m',
-          },
-          control: 'PBAnumber1',
-          validators: ['pbaNumberPattern', 'pbaNumberMaxLength', 'pbaNumberMinLength'],
-          validationError: {
-            value: 'Enter a valid PBA number',
-            controlId: 'PBAnumber1',
+        fieldset: [
+          {
+            legend: {
+              text: 'What\'s the Payment by Account (PBA) number for your organisation?',
```

Comment:
Given the fields below, this phrasing seems off - "What's" is short for "What is", which is a question about a single item, yet there are two PBA fields below. This should be re-phrased.

Reply:
rectified